File: C++/MathematicalPrograms/GramsToKilograms.cpp
// Convert grams to kilograms
#include <iostream>
#include <string>

using namespace std;

int main() {
	double grams;
	double kilograms;
	cout << "Enter the Measurement in Grams";
	cin >> grams;
	kilograms = grams/1000;
	cout << "Weight in kilograms = " << kilograms << " kg";
}

File: Java/MathematicalPrograms/GramsToKilograms.java
// Convert grams to kilograms
import java.util.Scanner;
public class GramsToKilograms {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the Measurement in Grams");
		float grammeasure = myObj.nextFloat();
		float kilogrammeasure = grammeasure/1000;
		System.out.println("Weight in kilograms = " + kilogrammeasure + " kg");
	}
}
		 